src/store/reducers/movies.ts

```diff
@@ -76,16 +76,18 @@ const movies = createSlice({
 export const { addMoviesState, addMovies, pushMovie, putMovie, sortMovies  } = movies.actions
 export const { addMovie, setShowHeader, setOpenForm, setEditStatus } = movie.actions
 
+export const selectNotEmptyListMovies = (state: RootState) => state.movies.data.length > 0
```

Comment:
I suppose it should be renamed. Cause right now it seems that this function return some **movies** instead of **boolean**. 
```js
export const isNotEmptyListMovies = (state: RootState) => !!selectMovies(state).length
```

Reply:
made edits in task 9